client/electron/index.ts

```diff
@@ -499,19 +499,19 @@ function main() {
     mainWindow?.webContents.send('outline-ipc-push-clipboard');
   });
 
-  // This IPC handler allows the renderer process to call Go API functions exposed by the backend.
+  // This IPC handler allows the renderer process to call functions exposed by the backend.
   // It takes two arguments:
-  //   - api: The name of the Go API function to call.
-  //   - input: A string representing the input data to the Go function.
+  //   - method: The name of the method to call.
+  //   - params: A string representing the input data to the function.
   //
   // The handler returns the output string from the Go function if successful.
   // Both the input string and output string need to be interpreted by the renderer process according
   // to the specific API being called.
-  // If Go function encounters an error, it throws an Error that can be parsed by the `PlatformError`.
+  // If the function encounters an error, it throws an Error that can be parsed by the `PlatformError`.
   ipcMain.handle(
-    'outline-ipc-invoke-go-api',
-    (_, api: GoApiName, input: string): Promise<string> =>
-      invokeGoApi(api, input)
+    'outline-ipc-invoke-method',
+    async (_, method: string, params: string): Promise<string> =>
+      await invokeMethod(method, params)
```

Comment:
*nit* We can remove this `async`/`await` pair, this also saves some resource (no need for Node to create a `Promise`).

```suggestion
    (_, method: string, params: string): Promise<string> =>
     invokeMethod(method, params)
```

Reply:
Done